parking_permits/services/traficom.py

```diff
@@ -171,7 +180,7 @@ def fetch_vehicle_details(self, registration_number):
             "vehicle_class": vehicle_class,
             "manufacturer": vehicle_manufacturer.text,
             "model": vehicle_model.text if vehicle_model is not None else "",
-            "weight": int(weight.text) if weight else 0,
```

Comment:
is this weight value not already converted to `int` further up?

Reply:
This was moved to earlier stage.